.github/workflows/ci.yaml

```diff
@@ -8,21 +8,44 @@ on:
     branches:
       - master
 
+defaults:
+  run:
+    shell: 'bash -o errexit -o nounset -o pipefail {0}'
+
 jobs:
   ci:
     runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        version: [2.5.23, 3.0.19, 4.0.15, 5.0.0-alpha-2]
     steps:
       - name: checkout
-        uses: actions/checkout@v2
+        uses: actions/checkout@v3
       - name: jdk setup
-        uses: actions/setup-java@v1
+        uses: actions/setup-java@v3
         with:
+          distribution: 'temurin'
           java-version: 11
           java-package: jdk
           architecture: x64
```

Comment:
This is the default

Reply:
Oh, I must have mis-read. I thought it was Zulu. I don't have a strong preference here.

Reply:
No I meant arch: x64 is the default

Reply:
No I meant arch: x64 is the default

Reply:
Ah, gotcha. I had that from back in the day for whatever reason. (possibly no reason).